apis/nucleus/src/viz.js

```diff
@@ -237,6 +272,28 @@ export default function viz({
       await rendered;
       return cellRef.current.takeSnapshot();
     },
+    // ===== undocumented experimental API - use at own risk ======
+    /**
+     *  valid types: viewData, cssScaling, snapshot, exportData, exploration
+     *  questionable types: supportRefresh, quickMobile, fullscreen
+     *  deprecated?: sharing
+     *
+     */
+    support(type) {
+      if (type === 'viewData' && viewDataObjectId !== undefined) {
+        return true;
+      }
```

Comment:
why only for type viewData?

there may be cases where we want more types to return support for the original type even when data view is toggle on. (maybe needed for download in hover menu, but not sure as I have not found any object type that support data view but not download)

Reply:
hmm, depends on how its used. in general I'd think you want to get the support response of the original chart always. But it also doesn't make much sense to check most things on the flipped chart.
The way its structured in nebula with toggleDataView makes it a bit tricky to return the original support object, would have to store it. Right now I'm not sure what the expected outcome is, so best leave it as is, and check back once we start using it and before adding it to the public API.

Reply:
consider if we should make it an error to call support in data view mode to prevent anyone from depending on it and make it more open for later changes

Reply:
a work around is maybe just return `viewDataObjectId != undefined`

Reply:
i think an error is good to begin with, makes it easy to root out potential bugs instead of just returning always false for example.

Reply:
Throwing an error is actually a problem. Let say toggled to view data of a barchart and I jump to another object. The next time I am at the barchart again and press space this time the barchart is already in view data. Therefore this check throw an error  
`const shouldToggleDataView = !focusedObjectApi.viewDataToggled && focusedObjectApi.support("viewData");`
Instead of giving me `shouldToggleDataView` so I can go and toggleFocus instead. 
https://github.com/qlik-trial/nebula-containers/blob/1251cf75d12ebfc7e154bc36923ec530b1920ccf/packages/sn-layout-container/src/hooks/use-interactions.ts#L325

I still vote for returning  `viewDataObjectId != undefined` when it is already in viewData.